package.json

```diff
@@ -12,18 +12,22 @@
     "test": "env TS_NODE_PROJECT=\"tsconfig.testing.json\" mocha --recursive --extension ts --require ts-node/register test",
     "start": "yarn run start-dev",
     "start-dev": "webpack-dev-server --host 0.0.0.0 --config=configs/webpack/dev.js",
-    "start-prod": "yarn run build && node express.js"
+    "start-prod": "yarn run build && node express.js",
+    "generate-i18n": "ts-node i18n/generate.ts",
+    "build-i18n": "ts-node i18n/build.ts"
```

Comment:
If `i18n.json` isn't checked in, does `yarn run build-i18n` need to be added as a step in CD?